packages/cactus-plugin-satp-hermes/src/main/typescript/blo/admin/get-healthcheck-handler-service.ts

```diff
@@ -0,0 +1,41 @@
+import { GetStatusError } from "../../core/errors/satp-errors";
+import { HealthCheckResponse } from "../../generated/gateway-client/typescript-axios/api";
+import { Logger } from "@hyperledger/cactus-common";
+import { SATPManager } from "../../gol/satp-manager";
+
+export async function ExecuteGetHealthCheck(
+  logger: Logger,
+  manager: SATPManager,
+): Promise<HealthCheckResponse> {
+  const fnTag = `GetHeathCheckHandler`;
+  logger.info(`${fnTag}, Obtaining healthcheck`);
+
+  try {
+    const result = await GetHealthCheckService(logger, manager);
+    return result;
+  } catch (error) {
+    if (error instanceof GetStatusError) {
+      logger.error(`${fnTag}, Error getting status: ${error.message}`);
+      throw error;
+    } else {
+      logger.error(`${fnTag}, Unexpected error: ${error.message}`);
+      throw new Error("An unexpected error occurred while obtaining status.");
```

Comment:
@AndreAugusto11 Very very new thing but on the main branch (as of 4 days ago) now you can use these constructs for chaining errors with their causes:
https://github.com/hyperledger-cacti/cacti/pull/3593

I'm not sure if the current base branch you are sending this PR against is up to date with main that recently, but just for at least future reference wanted to make note that you could soon (or maybe already) use the `cause` property of the `Error` built-in type as shown in the PR description of the PR linked above.
Just raising awareness.

Reply:
This is a neat feature

Reply:
@RafaelAPB could you please confirm that this branch is not rebased with main? In that case, I woud defer this change to a future PR when merging/rebasing with main